export_events.py

```diff
@@ -23,10 +23,10 @@ def loadEvents(fname):
       stamp = int(w[:ix])
       str = w[ix+1:]
       events.append({'t':stamp, 's':str})
-  except Exception, e:
-    print '%s probably does not exist, setting empty events list.' % (fname, )
-    print 'error was:'
-    print e
+  except Exception as e:
+    print ('%s probably does not exist, setting empty events list.' % (fname, ))
```

Comment:
Don't use white space ["Immediately before the open parenthesis that starts the argument list of a function call:"](https://pep8.org/#pet-peeves):
```python
print('%s probably does not exist, setting empty events list.' % (fname, ))
```